controller/src/test/java/io/pravega/controller/store/stream/ZkStreamImplTest.java

```diff
@@ -17,7 +17,7 @@
 
 import java.time.Duration;
 
-public class ZKStreamTest extends StreamTestBase {
+public class ZkStreamImplTest extends StreamTestBase {
```

Comment:
There is not `ZKStreamImpl` class, so the name change is odd. If you have to rename to avoid it starting with `ZK`, then spell it out and say `ZooKeeper`. 

Also, I'm not sure what it is that this class is tenting. Can it be consolidated with anything else?

Reply:
This class is derived class for streamtestbase class.. it’s sibling class is inmemorystreamtest class..
Incidentally there was another test class called zkstteamtest which clashed with its case insensitive name hence the PR.

Reply:
i like the idea of changing it from ZKSteram to Zookeeper stream to disambiguate

Reply:
Renamed to `ZooKeeperStreamTest`.